data/campaigns/Descent_Into_Darkness/scenarios/06_Return_to_Parthyn.cfg

```diff
@@ -259,53 +279,181 @@
 
                 [message]
                     speaker=Malin Keshar
-                    message= _ "A spirit I raised with the arts taught to me by Darken Volk. With his help, I have slain two orcish chieftains!"
+                    message= _ "It is of little concern to you. It is merely the byproduct of the new magical arts that I have learned."
                 [/message]
 
                 [message]
                     speaker=Drogan
-                    message= _ "You allowed yourself to be corrupted by the foul taint of the necromancer? Better to face the orcish menace than even dabble in dark magic! People would have rebuilt the town, just as they always have in the past. Your father would be ashamed to see the end to which you’ve come."
+                    message= _ "It does not take a trained mage to realize that those ‘magical arts’ refer to necromancy! That creature stinks of death!"
                 [/message]
 
                 [message]
                     speaker=Malin Keshar
-                    message= _ "I doubt that we will see any orc raids this summer. The beasts will be too busy trying to regroup! Is this how you repay me?"
+                    message= _ "And why should you care about such things? Orcs march on Parthyn! I am here to help defend the town!"
+                [/message]
+
+                [message]
+                    speaker=Drogan
+                    message= _ "Better to face the orcish menace alone than to dabble in dark magic! I warned you about this before, yet you failed to heed my advice. Your father would be ashamed to see the end to which you’ve come."
+                [/message]
+
+                [message]
+                    speaker=Malin Keshar
+                    message= _ "How dare you bring my father into this, Drogan! I have fought with every fiber of my being against these foul orcs, and I have done it to protect my country and kinsmen. Is this how you repay me? With scorn and disdain?"
+                [/message]
+
+                [message]
+                    role=advisor
+                    message=_ "Master, do not listen to him! He is clearly blinded by his own prejudice! He jeopardizes the safety of Parthyn!"
                 [/message]
             [/else]
         [/if]
 
         [message]
-            speaker=Dela Keshar
-            message= _ "You are mistaken, brother. Even now the orcs camp north of the river."
+            speaker=Malin Keshar
+            message= _ "Silence from you."
+        [/message]
+
+        [message]
+            speaker=Malin Keshar
+            message= _ "Drogan! With my newfound powers, I have already eliminated two orc chieftains! If you will allow me to help, I will crush the one who comes here as well!"
```

Comment:
I count four orcish warlords on eliminated on screen (2 in S4, 2 in S5), plus the off-screen ones mentioned in S5's intro.

Reply:
I think it refers to the ones only in S5, at least that's how I read it in the original. I could always change it to "many", though.